openmetadata-spec/src/main/resources/json/schema/metadataIngestion/databaseServiceMetadataPipeline.json

```diff
@@ -44,6 +44,11 @@
           "description": "DBT cloud account Id",
           "type": "string"
         },
+        "dbtCloudProjectId": {
+          "title": "DBT Cloud Project Id",
+          "description": "In case of multiple projects in a DBT cloud account, specify the project's id from which you want to extract the DBT run artifacts",
+          "type": "string"
+        },
```

Comment:
do we need to take some action to avoid retro compatibility issues? can we have a default value?

Reply:
@nahuelverdugo by default if noting is passed into this is field it will store `None` on which I have added conditions. We cannot pass any other default string value since it will make the wrong API call

Reply:
would empty string work here as default?

Reply:
@ayush-shah empty string wont work since i cannot pass an empty string into the API

Reply:
What if we read a value already stored in DB or JSON? Would it fail?

Reply:
@nahuelverdugo can you elaborate the use case a bit more?

Reply:
Resolving, since it has been solved offline. It has already been tested with null vales from UI and CLI.